### PR TITLE
feat(skin): IOS-9684 update design tokens from mistica-design branch: production

### PR DESCRIPTION
**Pull request by tuentisre**
## 🎟️ **Jira ticket**
https://jira.tid.es/browse/IOS-9684

## 🥅 **What's the goal?**

Update tokens related to new-vivo brand for 24.1.

## 🚧 **How do we do it?**
1. Get the changes from mistica-design repo using our tool => https://github.com/Telefonica/mistica-ios/actions/runs/7582804433
    - It generated this PR
2. Tests failed (obviously) => https://github.com/Telefonica/mistica-ios/actions/runs/7582809783
    - Tests Failed: **68** failed, 0 skipped, 458 total
3. Execute the screenshot generation action => https://github.com/Telefonica/mistica-ios/actions/runs/7583943484
    - **68** png's updated

## 🧪 **How can I verify this?**
Please @yceballost could you check that the changes are expected? the screenshots look good to me
I can generate a Mistica Catalog and/or a Novum app with this update.
✅  The affected screenshots are related to new-vivo [only]

### CODE REVIEW

**Comment by amegias** [comment]
@yceballost should we do https://github.com/Telefonica/mistica-ios/pull/327/files#diff-fb8824eed7bb698d3e5561f72e0dadd757aa09e1839758a9a6cf57a22bf0f101R77 ?

File: Tests/MisticaTests/UI/__Snapshots__/IndeterminateStepperTests/testIndeterminateStepperDefaultState.with-vivoNew-style.png
Comment (yceballost) [inline Tests/MisticaTests/UI/__Snapshots__/IndeterminateStepperTests/testIndeterminateStepperDefaultState.with-vivoNew-style.png:1]: This one is wrong, using barTrack should be solved

Comment (amegias) [reply Tests/MisticaTests/UI/__Snapshots__/IndeterminateStepperTests/testIndeterminateStepperDefaultState.with-vivoNew-style.png:1]: could you check it again please?

**Comment by amegias** [comment]
> > @yceballost should we do [#327 (files)](https://github.com/Telefonica/mistica-ios/pull/327/files#diff-fb8824eed7bb698d3e5561f72e0dadd757aa09e1839758a9a6cf57a22bf0f101R77) ?
> 
> yes please, use barTrack token in progressBar / Stepper / Slider (I think slider is not included in mística ios)

Done, could you check it again pls?

**Review by pbartolome — APPROVED** [review]

**Review by yceballost — APPROVED** [review]

**Review by alejandroruizponce — APPROVED** [review]

**Comment by amegias** [comment]
Screenshots result are not updated (since the actions cannot be triggered from a bot action like comiting screenshots). Anyway, CI is passing https://github.com/Telefonica/mistica-ios/actions/runs/7624085949

**Pull request merged**